foundations/html_css/block-and-inline.md

```diff
@@ -19,6 +19,18 @@ Inline elements, however, do not start on a new line. They appear in line with w
 
 Inline-block elements behave like inline elements, but with block-style padding and margin. Inline-block is a useful tool to know about, but in practice, you'll probably end up reaching for flexbox more often if you're trying to line up a bunch of boxes. Flexbox will be covered in-depth in the next lesson.
 
+Here's a table comparing and contrasting `block`, `inline-block`, and `inline`:
+
+|                                                                    |       `block`      |   `inline-block`   |          `inline`         |
+|:------------------------------------------------------------------:|:------------------:|:------------------:|:-------------------------:|
```

Comment:
Table's a nice idea, we definitely don't use enough of them. But It'll be hard to tell the difference between the properties from table entries alone, the box model is a very visual thing. I think paring the table with a few codepens would be the best way to do it.

Would you like to put a few examples together?